bin/main.py

```diff
@@ -81,4 +91,8 @@ def process_delay(signal, value, delay):
 
 if __name__ == "__main__":
     main = Main()
-    run()
+    if len(sys.argv) > 1 and sys.argv[1] == '--ipc':
```

Comment:
This is too trivial for argument parsing, you'll have to coordinate with fagan here since he'll be making the program accept the configuration file as a parameter. Also, you need to let the user specify which module to load. Either the user does not specify a module, and then you should use the default send/receive functions (that may be from a module - let's say, a stdin/stdout module), or they specify an ipc module that is loaded.

For instance, this should load and assign the zeromq receive and send functions to be used by the main program:

./vsm --ipc-module=zeromq configuration.yaml

This should either have default functions being used or loading a default module, which uses stdin/stdout:

./vsm configuration.yaml